package.json

```diff
@@ -25,7 +25,7 @@
   },
   "devDependencies": {
     "concurrently": "^4.0.1",
-    "cypress": "6.2.0",
+    "cypress": "^7.7.0",
```

Comment:
you should never do this as a part of your ticket you are working on
trust me, bumping of packages version usually needs a lot of time and effort to be done
in our case: small app, small test suite - it is okay, but try to always create separate PR for this

Reply:
Discussed on call